packages/expo-dev-launcher/bundle/hooks/useDevMenuSettings.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
```

Comment:
I see what you want to do here, but I think we can do it better. Your current solution looks like `expo-dev-launcher` is depending on `expo-dev-menu`.  We can get settings via `expo-dev-menu-interface` and reexported functions through the dev-launcher internal module (if dev-menu is present). I have one more idea. Maybe we could set `expo-dev-menu` as an optional dependency of the `expo-dev-launcher and export functions to edit setting through the dev menu public module. What do you think? Of course, we can land it, and later we can do the refactor.